lhotse/recipes/ami.py

```diff
@@ -84,15 +177,59 @@ def download(
                 wav_dir.mkdir(parents=True, exist_ok=True)
                 wav_path = wav_dir / wav_name
                 if force_download or not wav_path.is_file():
-                    urllib.request.urlretrieve(wav_url, filename=wav_path)
+                    urlretrieve_progress(wav_url, filename=wav_path, desc=f'Downloading {wav_name}')
+        elif mic == 'ihm-mix':
+            wav_name = f'{item}.Mix-Headset.wav'
+            wav_url = f'{url}/AMICorpusMirror/amicorpus/{item}/audio/{wav_name}'
+            wav_dir = target_dir / 'wav_db' / item / 'audio'
+            wav_dir.mkdir(parents=True, exist_ok=True)
+            wav_path = wav_dir / wav_name
+            if force_download or not wav_path.is_file():
+                urlretrieve_progress(wav_url, filename=wav_path, desc=f'Downloading {wav_name}')
+        elif mic == 'sdm':
+            wav_name = f'{item}.Array1-01.wav'
+            wav_url = f'{url}/AMICorpusMirror/amicorpus/{item}/audio/{wav_name}'
+            wav_dir = target_dir / 'wav_db' / item / 'audio'
+            wav_dir.mkdir(parents=True, exist_ok=True)
+            wav_path = wav_dir / wav_name
+            if force_download or not wav_path.is_file():
+                urlretrieve_progress(wav_url, filename=wav_path, desc=f'Downloading {wav_name}')
+        elif mic == 'mdm':
+            for array in MDM_ARRAYS:
+                for channel in MDM_CHANNELS:
+                    wav_name = f'{item}.{array}-{channel}.wav'
+                    wav_url = f'{url}/AMICorpusMirror/amicorpus/{item}/audio/{wav_name}'
+                    wav_dir = target_dir / 'wav_db' / item / 'audio'
+                    wav_dir.mkdir(parents=True, exist_ok=True)
+                    wav_path = wav_dir / wav_name
+                    if force_download or not wav_path.is_file():
+                        urlretrieve_progress(wav_url, filename=wav_path, desc=f'Downloading {wav_name}')
 
+def download(
```

Comment:
Can we rename to `download_ami`? I'll be updating other recipes in a similar way.